apps/admin-x-activitypub/src/routes.ts

```diff
@@ -0,0 +1,43 @@
+import Inbox from '@views/Inbox';
+import Notifications from '@views/Notifications';
+import Profile from '@views/Profile';
+import Search from '@views/Search';
+import {RouteObject} from '@tryghost/admin-x-framework';
+
+export const APP_ROUTE_PREFIX = '/activitypub';
+
+export const routes: RouteObject[] = [
+    {
+        path: '',
+        Component: Inbox
+    },
+    {
+        path: 'inbox',
+        Component: Inbox
+    },
+    {
+        path: 'feed',
+        Component: Inbox
+    },
+    {
+        path: 'search',
+        Component: Search
+    },
+    {
+        path: 'notifications',
+        Component: Notifications
+    },
+    {
+        path: 'profile',
+        Component: Profile
+    }
+];
+
+export const ROUTE_TITLES: Record<string, string> = {
```

Comment:
It would be cool if we can define this inline with the `routes` so we know that there is one for each path